generator/insn_vdvs2.go

```diff
@@ -9,8 +9,8 @@ import (
 )
 
 func (i *Insn) genCodeVdVs2(pos int) []string {
-	zvkg_insn := strings.HasPrefix(i.Name, "vg")
-	sews := iff(zvkg_insn, []SEW{32}, allSEWs)
+	sew32Only_insn := strings.HasPrefix(i.Name, "vg") || strings.HasPrefix(i.Name, "vsm4")
```

Comment:
Overall, this PR looks good to me, but we don't use underscore in variable names, please use the camel style, i.e. s/sew32Only_insn/sew32OnlyInsn/, and the same for other places.